### PR TITLE
Pr/installation delta action

### PR DESCRIPTION
**Pull request by MethodenMann**

### CODE REVIEW

File: packages/installation-delta-action/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+## Embedding
+
+React-registry name: `installation-delta-action`
+
+### Input parameters
+
+| Name      | Mandatory | Description
+|-----------|:---------:|-------------
+| `keys`    | x         | Array of installation keys
+| `issueUrl`| x         | Url to jiira
```
Comment (rzueger) [inline packages/installation-delta-action/README.md]: Jira

File: packages/installation-delta-action/src/components/CommitInfo/CommitInfo.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Typography as T, FormattedValue} from 'tocco-ui'
+import styled from 'styled-components'
+
+const TimeWrapper = styled.span`
+  time {
+  overflow: visible !important;
```
Comment (rzueger) [inline packages/installation-delta-action/src/components/CommitInfo/CommitInfo.js]: (human linter: indent)

File: packages/installation-delta-action/src/components/IssueInfos/IssueInfos.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      label={`${issue.key} (${issue.status})`}
+    />
+  </Typography.P>
+)
+
+IssueInfos.propTypes = {
+  issues: PropTypes.arrayOf(PropTypes.shape({
+    key: PropTypes.string,
+    status: PropTypes.string
+  }))
```
Comment (rzueger) [inline packages/installation-delta-action/src/components/IssueInfos/IssueInfos.js]: isRequired

File: packages/installation-delta-action/src/components/IssueInfos/IssueInfos.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Link, Typography} from 'tocco-ui'
+
+const IssueInfos = ({issues, issueUrl}) => issues.map((issue, idx) =>
```
Comment (rzueger) [inline packages/installation-delta-action/src/components/IssueInfos/IssueInfos.js]: `issueUrl` missing in propTypes

File: packages/installation-delta-action/README.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+## Embedding
+
+React-registry name: `installation-delta-action`
+
+### Input parameters
+
+| Name      | Mandatory | Description
+|-----------|:---------:|-------------
+| `keys`    | x         | Array of installation keys
```
Comment (rzueger) [inline packages/installation-delta-action/README.md]: must contain exactly 2 keys (?). would be worth to mention IMO.

File: packages/installation-delta-action/src/components/Header/Header.js [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+    const options = {credentials: 'include'}
+    const url = `${__BACKEND_URL__}/nice2/rest/entities/Installation/${key}?relations=!&_fields=!`
+    const response = await fetch(url, options)
+    return response.json()
+  }
+
+  const fetchLabels = async() => {
+    const [instanceInstallation, deltaInstallation] = await Promise.all(
+      [getFetchRequest(keys[0]),
+        getFetchRequest(keys[1])]
```
Comment (rzueger) [inline packages/installation-delta-action/src/components/Header/Header.js]: maybe throw meaningful error if `keys` does not contain 2 keys or use custom proptype function for validation - or at least document it in readme.md

File: packages/installation-delta-action/src/components/InstallationDelta/InstallationDelta.js [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  border: 1px solid #888888;
+  background: ${props => props.isOdd ? '#F5F5F5' : ''};
+`
+
+const InstallationDelta = ({keys}) => {
+  const [delta, setDelta] = useState(null)
+
+  const fetchData = async() => {
+    const options = {credentials: 'include'}
+    const url = `${__BACKEND_URL__}/nice2/rest/tocco/commit-info/installation/${keys[0]}/delta?installation=${keys[1]}`
```
Comment (rzueger) [inline packages/installation-delta-action/src/components/InstallationDelta/InstallationDelta.js]: see comment in Header.js

File: packages/installation-delta-action/src/components/Header/Header.js (hunk 6 above)
Comment (MethodenMann) [reply packages/installation-delta-action/src/components/Header/Header.js]: Thought about it before but this check happens in nice. imo not worth the effort. But i will add it to the readme.

**Pull request merged**